addons/l10n_cn/data/template/account.tax-cn.csv

```diff
@@ -3,11 +3,11 @@
 "","","","","","","","","","tax","invoice","l10n_cn_2221",""
 "","","","","","","","","","base","refund","",""
 "","","","","","","","","","tax","refund","l10n_cn_2221",""
-"l10n_cn_sales_included_9","9% INC","VAT 9%（included)","9%","9.0","percent","sale","True","l10n_cn_tax_group_vat_9","base","invoice","","税收9％（含)"
+"l10n_cn_sales_included_9","9% INC","VAT 9%(included)","9%","9.0","percent","sale","True","l10n_cn_tax_group_vat_9","base","invoice","","税收9%(含)"

```

Comment:
Small detail but why do you remove the space there? Also you do it for 9 & 6 but not 13.

Reply:
Ah this was not a space. Originally they use "（" which is a Chinese open bracket. Also they used "％"  and "%" inconsistently. It's a common problem given they use English and Chinese input at the same time.
But yea I think they used the Chinese open bracket to make indentation. Let me add a space to all instead.

And yes let me ask Crystal again later.
Thanks.